docker-compose.yml

```diff
@@ -7,16 +7,17 @@ networks:
 services:
   web:
     container_name: grant-web
-    image: grant-web-builder 
+    image: grant-web-builder
     build:
       context: .
-      target: builder 
+      target: builder
     ports:
       - "3333:3333"
     environment:
       - "DATABASE_URL=postgres://grants:password@postgres/grants?sslmode=disable"
       - "LEDGER_SERVER=https://ledger-staging.mercury.basicattentiontoken.org"
       - "REDIS_URL=redis://redis:6379"
+      - "PG_MIGRATIONS=file:///src/migrations"
```

Comment:
`PG_MIGRATIONS` doesn't quality the expected format.

e.g. does `PG_MIGRATIONS` expect true/false to set whether migrations are performed? it doesn't but that's not clear from the current name whereas it would be for `PG_MIGRATIONS_URL`.

also, consider that currently the database URL is `DATABASE_URL` - perhaps `DATABASE_MIGRATIONS_URL` fits better